jest-puppeteer-custom-env.js

```diff
@@ -0,0 +1,53 @@
+// custom-environment.js
+const PuppeteerEnvironment = require('jest-environment-puppeteer');
+const fs = require('fs');
+
+// Configure
+const testDir                      = './tests'
+const imageSnapshotsDir            = testDir+'/__image_snapshots__';
```

Comment:
Path construction should better be done in a OS agnostic way:
```
const path = require('path');
...
path.join(testDir, '__image_snapshots__');
```

Applicable to multiple places in this file...